src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -53,7 +53,13 @@
         <DataSource Persistence="ProjectFile" PersistedName="TargetFileName" SourceOfDefaultValue="AfterContext" />
       </StringProperty.DataSource>
     </StringProperty>
-  
+
+    <BoolProperty Name="AutoGenerateBindingRedirects" DisplayName="Auto-Generate Binding Redirects">
```

Comment:
Can you (or @davkean) explain why this has to be changed here and in `ConfigurationGeneral.xaml`?

Reply:
The change in ConfigurationGeneral.xaml is not needed and has been backed out.